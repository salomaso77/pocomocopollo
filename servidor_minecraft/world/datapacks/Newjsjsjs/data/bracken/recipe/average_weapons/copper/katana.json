{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		"  #",
		" # ",
		"/  "
	],
	"key": {
		"#": "minecraft:copper_ingot",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:stone_sword",
		"components": {
			"minecraft:max_damage": 175,
			"minecraft:item_name": "Copper Katana",
			"minecraft:item_model": "bracken:average_weapons/copper/katana",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0.25,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 4,
	"factor": 4
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_katana"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "5.5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+2.5% Movement Speed","color":"dark_green","italic":false},				
				{"translate": "Projectile Deflection","color": "aqua","italic": false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:copper_katana",
					"amount": 4.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:copper_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:copper_katana",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}