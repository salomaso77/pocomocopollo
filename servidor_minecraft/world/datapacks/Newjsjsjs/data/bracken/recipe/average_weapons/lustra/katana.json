{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:golden_sword",
	"addition": "minecraft:sheep_spawn_egg",
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:max_damage": 3000,
			"minecraft:enchantment_glint_override": true,
			"minecraft:item_name": "Lustra Katana",
			"minecraft:item_model": "bracken:average_weapons/lustra/katana",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0.25,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 4,
	"factor": 4
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_katana"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "4 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+2.5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Projectile Deflection","color": "aqua","italic": false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:lustra_katana",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:lustra_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:lustra_katana",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}