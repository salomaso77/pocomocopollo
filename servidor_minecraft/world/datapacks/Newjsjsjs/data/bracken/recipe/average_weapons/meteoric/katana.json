{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		"  #",
		" # ",
		"/  "
	],
	"key": {
		"#": "minecraft:netherite_scrap",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Meteoric Katana",
			"minecraft:item_model": "bracken:average_weapons/meteoric/katana",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0.25,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 4,
	"factor": 4
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_katana"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "8 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+2.5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Projectile Deflection","color": "aqua","italic": false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:meteoric_katana",
					"amount": 7,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:nmeteoric_katana",
					"amount": -3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:meteoric_katana",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}