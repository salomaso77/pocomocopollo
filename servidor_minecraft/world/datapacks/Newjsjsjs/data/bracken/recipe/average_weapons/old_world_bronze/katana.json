{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:cat_spawn_egg",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Old World Bronze Katana",
			"minecraft:item_model": "bracken:average_weapons/old_world_bronze/katana",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0.25,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 4,
	"factor": 4
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_katana"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "8.5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+2.5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Projectile Deflection","color": "aqua","italic": false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:netherite_katana",
					"amount": 7.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:netherite_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:netherite_katana",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
				
		}
	}
}