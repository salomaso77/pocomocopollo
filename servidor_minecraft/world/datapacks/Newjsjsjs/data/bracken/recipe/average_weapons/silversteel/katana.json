{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:wolf_spawn_egg",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Silversteel Katana",
			"minecraft:enchantments": {
				"bracken:silvered_fix": 1
			},
			"minecraft:enchantment_glint_override": false,
			"minecraft:item_model": "bracken:average_weapons/silversteel/katana",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0.25,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 4,
	"factor": 4
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_katana"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "9 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+2.5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Projectile Deflection","color": "aqua","italic": false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:silversteel_katana",
					"amount": 8,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:silversteel_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:silversteel_katana",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}