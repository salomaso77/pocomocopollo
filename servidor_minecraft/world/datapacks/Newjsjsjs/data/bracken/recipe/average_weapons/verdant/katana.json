{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:wayfinder_armor_trim_smithing_template",
	"base": "minecraft:wooden_sword",
	"addition": "minecraft:creeper_spawn_egg",
	"result": {
		"id": "minecraft:wooden_sword",
		"components": {
			"minecraft:max_damage": 500,
			"minecraft:item_name": "Verdant Katana",
			"minecraft:item_model": "bracken:average_weapons/verdant/katana",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0.25,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 4,
	"factor": 4
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_katana"
},
			"minecraft:enchantments": {
				"bracken:infused_mending_fix": 1,
				"bracken:poisoning_fix": 1
			},
			"minecraft:enchantment_glint_override": false,
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "5.5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+2.5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Projectile Deflection","color": "aqua","italic": false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:verdant_katana",
					"amount": 4.5,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:verdant_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:verdant_katana",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}