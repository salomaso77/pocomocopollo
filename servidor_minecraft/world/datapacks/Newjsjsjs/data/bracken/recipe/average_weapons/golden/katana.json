{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		"  #",
		" # ",
		"/  "
	],
	"key": {
		"#": "minecraft:gold_ingot",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:item_name": "Golden Katana",
			"minecraft:item_model": "bracken:average_weapons/golden/katana",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0.25,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 4,
	"factor": 4
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_katana"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "4 Attack Damage","color":"dark_green","italic":false},
				{"translate": "1.6 Attack Speed","color":"dark_green","italic":false},
				{"translate": "+2.5% Movement Speed","color":"dark_green","italic":false},
				{"translate": "Projectile Deflection","color": "aqua","italic": false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:golden_katana",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:golden_katana",
					"amount": -2.4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:movement_speed",
					"id": "bracken:golden_katana",
					"amount": 0.025,
					"operation": "add_value",
					"slot": "hand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}